app/views/users/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="well">
+<% @users.each do |user| %>
+  <p>
+  <%= user.email %> 
+  <% if current_user.admin? %>
```

Comment:
looks like this html can be separated into partial.
Also, it is a bad practice to have mixed hash styles: `:class => 'btn btn-xs btn-danger', method: :delete`